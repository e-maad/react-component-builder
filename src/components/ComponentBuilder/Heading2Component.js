import React from 'react';

const Heading2Component = ({ text, id }) => {
    return <h2 id={id} className='heading2-component'>{text}</h2>
}

export default Heading2Component;