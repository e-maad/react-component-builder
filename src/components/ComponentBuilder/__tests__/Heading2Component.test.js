import React from 'react';
import Heading2Component from '../Heading2Component';


const wrapper = shallow(<Heading2Component />);
describe('Heading2 Component', () => {

  it('match the snapshot', () => {
    const wrapper = renderer.create(<Heading2Component />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });

  it('base element should be present', () => {
    expect(wrapper.find('.heading2-component').length).toBe(1);
  });
});