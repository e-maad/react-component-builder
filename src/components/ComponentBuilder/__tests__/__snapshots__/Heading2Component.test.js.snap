// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading2 Component match the snapshot 1`] = `
<h2
  className="heading2-component"
/>
`;
